HTML/Rebooting M-OS.html

```diff
@@ -21,24 +21,24 @@
             <div class="page page-bg-01">
                 <div class="page-header">
                     <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
-                    <span class="page-header-section-title">Rebooting M-OS</span>
+                    <span class="page-header-section-title">Rebooting Mystery OS</span>
                 </div>
                 <div class="page-content">
                     <img src="img/Component/Rebooting M-OS.svg" class="diagram">
-                    <h2>On the Subject of Rebooting M-OS</h2>
+                    <h2>On the Subject of Rebooting Mystery OS</h2>
                     <p class="flavour-text">Make a big reboom!</p>
                     <p>
                         This module use an unknown operation system. We're calling it "M-OS". At the start of each bomb,
                         this OS will display a fatal error message. However, there is just a moving square instead of any text. The square will cycle through 4 of 7 possible colors: red, green, blue, yellow, cyan, magenta and white.
-                        To defuse it, simply reboot M-OS. Unfortunately, the bomb's time greatly affects M-OS and turns it into an <i>advanced version</i>, which is harder to reboot.
```

Comment:
Most of the line changes still reverts the grammatical errors.

Reply:
The same thing is currently on the repo.